configs/other/server-environment-startup.json

```diff
@@ -9,6 +9,6 @@
         "serverPort": 3375,
         "serverConfig": "http://localhost:3375/configs/openfin/manifest-local.json",
         "clientRoute": "http://localhost:3375/",
-        "channelAdapter": "e2o"
+        "channelAdapter": "openfin"
```

Comment:
This change shouldn't be in this PR either.

Reply:
I think the issue is that you are merging with master rather than develop, but the PR is against develop, so you are picking up changes that shouldn't be there.